version.go

```diff
@@ -2,3 +2,12 @@ package egoscale
 
 // Version of the library
 const Version = "0.21.0"
+
+const (
+	computeAPIVersion = "v1"
+	// ComputeAPIEndpoint is the Exoscale Compute endpoint with the latest API version.
```

Comment:
Let's assume clients will always use the up-to-date version, and if they want the N+1 or N-1 version they'll have to build their own endpoint string.

```suggestion
	// ComputeAPIEndpoint represents the Exoscale Compute API endpoint.
```